guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -9,15 +9,15 @@ To create a file type repository in a directory on the base system where {Projec
 
 .Prerequisites
 ifdef::katello,orcharhino[]
-* You have a server running {EL} 8 registered to your {Project}.
+* You have a server running {EL} 9 registered to your {Project}.
```

Comment:
AFAIK, you can still run Foreman/Katello on EL8: https://yum.theforeman.org/releases/3.12/el8/x86_64/

```suggestion
* You have a server running {EL} 9 or {EL} 8 registered to your {Project}.
```